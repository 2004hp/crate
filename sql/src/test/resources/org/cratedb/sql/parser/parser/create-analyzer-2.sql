create analyzer "schema_name"."analyzer_name" with (
    tokenizer "ultimateThruth" with (
        answer=42
    )
)