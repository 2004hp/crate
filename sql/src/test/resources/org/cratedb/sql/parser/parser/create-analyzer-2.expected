org.cratedb.sql.parser.parser.CreateAnalyzerNode@277b65b1
name: schema_name.analyzer_name
statementType: CREATE ANALYZER
	
	org.cratedb.sql.parser.parser.AnalyzerElements@b24acca

	Tokenizer:	
		org.cratedb.sql.parser.parser.NamedNodeWithOptionalProperties@5b420a67
		ultimateThruth
		Properties: 			
			org.cratedb.sql.parser.parser.GenericProperties@4cb91eff
			{answer=value: 42
			type: INTEGER NOT NULL
			}
	TokenFilters:
		org.cratedb.sql.parser.parser.TokenFilterList@5effaa35

	CharFilters:
		org.cratedb.sql.parser.parser.CharFilterList@62ffae4c
